config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -17,7 +18,8 @@ presubmits:
     path_alias: "sigs.k8s.io/cluster-api-provider-azure"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-1.12
+      - image: gcr.io/k8s-testimages/kubekins-e2e:latest-1.14
```

Comment:
should we use images built from master, e.g. gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master? Most packages are using this image instead of latest-1.14 tag.

Reply:
@feiskyer -- I bumped to `latest-master` in #12224